_maps/RandomRuins/SpaceRuins/abandonedzoo.dmm

```diff
@@ -690,6 +690,21 @@
 "KN" = (
 /turf/template_noop,
 /area/space/nearstation)
+"UK" = (
+/obj/effect/turf_decal/tile/green{
+	dir = 1
+	},
+/obj/effect/turf_decal/tile/green,
+/obj/effect/turf_decal/tile/green{
+	dir = 4
+	},
+/obj/effect/turf_decal/tile/green{
+	dir = 8
+	},
+/obj/structure/cable,
+/obj/machinery/spaceship_navigation_beacon,
```

Comment:
I feel like it shouldn't be mapped in at start anywhere

Reply:
i remove all map edits after public test